python_modules/libraries/dagster-tableau/dagster_tableau/asset_utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import Sequence, Tuple
+
+from dagster import AssetSpec
+
+from dagster_tableau.translator import TableauTagSet
+
+
+def parse_tableau_external_and_materializable_asset_specs(
+    specs: Sequence[AssetSpec],
+) -> Tuple[Sequence[AssetSpec], Sequence[AssetSpec]]:
```

Comment:
could we return a NamedTuple instead, so that the return value here is self-documenting? would also be useful to add docstring if we want this to be user-visible

Reply:
That makes sense - I implemented a custom NamedTuple in [0838c35](https://github.com/dagster-io/dagster/pull/25670/commits/0838c352b94b274cfd99a6a213b0337a872c62d5).

The docstrings are in the subsequent docs PR